### PR TITLE
Basic code setup for Adobe-Students pages

### PR DESCRIPTION
**Pull request by sivasadobe**
* Basic setup from AEM Boilerplate

Resolves: [MWPW-156206](https://jira.corp.adobe.com/browse/MWPW-156206)

**Test URLs:**
- Before: https://main--adobe-students--adobecom.hlx.live/creativecloud/buy/students?martech=off
- After: https://stage--adobe-students--adobecom.hlx.live/creativecloud/buy/students?martech=off

### CODE REVIEW

File: .kodiak/config.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+notifications:
+  jira:
+    default:
+      project: MWPW # Mandatory
+      filters:
+        include:
+          risk_rating: R5
+      fields:
+        assignee:
+          name: blaishram
```
Comment (aishwaryamathuria) [inline .kodiak/config.yaml]: update asignees and watchers

File: .kodiak/config.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+version: 1.0
+
+snow:
+  - id: 546348 # Milo Genuine https://adobe.service-now.com/service_registry_portal.do#/service/546348
```
Comment (aishwaryamathuria) [inline .kodiak/config.yaml]: Update snow id

File: .github/codeql/codeql-config.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+name: "CC CodeQL Config"
```
Comment (aishwaryamathuria) [inline .github/codeql/codeql-config.yml]: Updae name

File: CODEOWNERS [hunk 4]
```diff
@@ -0,0 +1 @@
+* @adobecom/cc-code-owners @salonijain3 @bandana147 @amitbikram @raga-adbe-gh
```
Comment (aishwaryamathuria) [inline CODEOWNERS]: Update codeowners

File: adobe-students/styles/styles.css [hunk 5]
```diff
@@ -0,0 +1,16 @@
+ .reading-width {
```
Comment (aishwaryamathuria) [inline adobe-students/styles/styles.css:1]: remove this file

File: adobe-students/scripts/utils.js [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+      const branch = new URLSearchParams(window.location.search).get('milolibs') || 'main';
+      if (branch === 'local') { libs = 'http://localhost:6456/libs'; return libs; }
+      if (branch.indexOf('--') > -1) { libs = `https://${branch}.hlx.live/libs`; return libs; }
+      libs = `https://${branch}--milo--adobecom.hlx.live/libs`;
+      return libs;
+    }, () => libs,
+  ];
+})();
+
+const miloLibs = setLibs('/libs');
```
Comment (aishwaryamathuria) [inline adobe-students/scripts/utils.js]: we dont need the cc specific imports and functions. We can have this basic from milo-college

File: adobe-students/scripts/scripts.js [hunk 7]
```diff
@@ -0,0 +1,186 @@
...
+  locales,
+  geoRouting: 'on',
+  prodDomains: ['www.adobe.com', 'helpx.adobe.com', 'business.adobe.com', 'www.adobe-students.com'],
+  decorateArea,
+  stage: {
+    marTechUrl: 'https://assets.adobedtm.com/d4d114c60e50/a0e989131fd5/launch-2c94beadc94f-development.min.js',
+    edgeConfigId: '8d2805dd-85bf-4748-82eb-f99fdad117a6',
+    pdfViewerClientId: '9f7f19a46bd542e2b8548411e51eb4d4',
+    pdfViewerReportSuite: 'adbadobenonacdcqa',
+    psUrl: 'https://stage.photoshop.adobe.com',
```
Comment (aishwaryamathuria) [inline adobe-students/scripts/scripts.js]: remove the cc config

File: adobe-students/scripts/scripts.js [hunk 8]
```diff
@@ -0,0 +1,186 @@
...
+  live: {
+    pdfViewerClientId: 'a26c77a2effb4c4aaa71e7c46385e0ed',
+    pdfViewerReportSuite: 'adbadobenonacdcqa',
+  },
+  prod: {
+    marTechUrl: 'https://assets.adobedtm.com/d4d114c60e50/a0e989131fd5/launch-5dd5dd2177e6.min.js',
+    edgeConfigId: '2cba807b-7430-41ae-9aac-db2b0da742d5',
+    pdfViewerClientId: '409019ebd2d546c0be1a0b5a61fe65df',
+    pdfViewerReportSuite: 'adbadobenonacdcprod',
+    psUrl: 'https://photoshop.adobe.com',
```
Comment (aishwaryamathuria) [inline adobe-students/scripts/scripts.js]: remove cc config

File: .kodiak/config.yaml [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+notifications:
+  jira:
+    default:
+      project: MWPW # Mandatory
+      filters:
+        include:
+          risk_rating: R5
+      fields:
+        assignee:
+          name: sumanh
```
Comment (aishwaryamathuria) [inline .kodiak/config.yaml]: Change this ldap

File: .kodiak/config.yaml [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+        customfield_11800: MWPW-140779 #epic link
+        customfield_12900: 
+          value: Brahmos
+        watchers:
+          - casalino
+          - jmichnow
+          - mauchley
+          - saljain
+          - mathuria
+          - dstrong
```
Comment (aishwaryamathuria) [inline .kodiak/config.yaml]: Change the watchers list

**Review by aishwaryamathuria — APPROVED** [review]

**Review by salonijain3 — APPROVED** [review]

**Pull request merged**